Formula/hyeong.rb

```diff
@@ -0,0 +1,15 @@
+class Hyeong < Formula
+  desc "Hyeo-ung Programming Language Compiler in Rust (beta release)"
+  homepage "https://github.com/buttercrab/hyeo-ung-lang"
+  url "https://github.com/buttercrab/hyeo-ung-lang/releases/download/v0.1.0/hyeong-0.1.0-darwin-x86-64.tar.gz"
```

Comment:
Formulae need to build from source: https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae